ISSUE_TEMPLATE.md

```diff
@@ -4,8 +4,10 @@
 - [ ] Issue Title is in form `Algo/DS Name [Language]`.
 - [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
 - [ ] This Algo/DS is worth adding to this repository.
+- [ ] I ensure that I'm not assigned to 2 or more issues.
```

Comment:
Shouldn't this be `I ensure that I'm not assigned to more than 2 issues.` as we allow 2 issues at a time?